main.cpp

```diff
@@ -593,16 +627,25 @@ int main()
     std::cout << "Initial value of it: " << it << std::endl;
     // --------
     std::cout << "Use of function concatenation (mixed type arguments) " << std::endl;
-    std::cout << "New value of dt = (dt * it) / 5.0f + ft = " << (dt.multiply( it ).divide(static_cast<double>(5.0f) ).add(static_cast<double>(ft) )) << std::endl;
+    dt*= it;
+    dt/= static_cast<double>(5.0f);
+    dt+= static_cast<double>(ft);
+    std::cout << "New value of dt = (dt * it) / 5.0f + ft = " << dt << std::endl;
 
     std::cout << "---------------------\n"<< std::endl; 
     
     // Intercept division by 0
     // --------
     std::cout << "Intercept division by 0 " << std::endl;
-    std::cout << "New value of it = it / 0 = " << it.divide(0) << std::endl;
-    std::cout << "New value of ft = ft / 0 = " << ft.divide(0) << std::endl;
-    std::cout << "New value of dt = dt / 0 = " << dt.divide(0) << std::endl;
+    std::cout << "New value of it = it / 0 = ";
+    it/= ( 0 );
+    std::cout << it << std::endl;
+    std::cout << "New value of ft = ft / 0 = ";
+    ft/= ( 0 );
+    std::cout << ft << std::endl;
+    std::cout << "New value of dt = dt / 0 = ";
+    dt/= ( 0 );
+    std::cout << dt << std::endl;
```

Comment:
Add some space around the operators here.
`it+=` `dt*=` etc are difficult to read.
Help yourself and make the code easier to read
- [ ] revise